static/app/components/replays/playerRelativeTime.tsx

```diff
@@ -12,9 +14,38 @@ const PlayerRelativeTime = ({relativeTime, timestamp}: Props) => {
     return <div />;
   }
 
-  return <Value>{showPlayerTime(timestamp, relativeTime)}</Value>;
+  return (
+    <Wrapper>
+      <Tooltip
+        title={getDateFormatted(timestamp)}
```

Comment:
Can we use the `<DateTime>` component here?

Reply:
Changed to `<DateTime />` component